aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,105 +41,73 @@ import { Example } from '../../layouts';
   <SimpleSignUpExample />
 </Example>
 
-## Guidance
+## When to use
 
-Forms provide a way to collect important data from a user. However, this data is only collected effectively when Forms are built in a way in which users are inclined to fill out in full.
+To efficiently capture information necessary for completing a task.
 
-### Usage
+## When not to use
 
-Forms don't need to be a tedious task for users to complete, as they often come to be. It is important to ease as much of this effort as possible when building a Form. Making Forms simple to complete leads to increase in form completion.
+Complicated processes where the user might need guidance at various stages, or when the form would be intimidatingly long. In these cases use a [Wizard](https://design-system.hpe.design/templates/wizard?q=error#guidance).
 
-**Follow these guidelines when creating a Form:**
 
-- Use as few fields as possible
-- Wrap all input elements within a FormField
-- Fill in default values for as many fields as possible
-- Reveal fields progressively so the user is not overwhelmed
-- Use clear, concise labels for each field
-- Use placeholder text as an added visual indication of expected input value format
+## Do's
 
-### Required vs. optional fields
+**Required fields**
+- Ask only for the required information. Keep the fields to a minimum to allow the user to finish their task as quickly as possible.
+- List only the most common field requirements below the relevant field 
+- Use asterisks for required fields when the form has more than two required fields and/or has optional fields. [See HPE UX research](https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=4988%3A3653).  
+- Asterisks for required fields should be the same color as label field 
+- If a field is optional, write "Optional" in italics next to the form field label
 
-- When possible, restrict forms only to fields that are required in order to keep them concise.
-- For [single form fields](#single-required-formfield) (ex: delete confirmation request), use an asterisk to denote that it is a "required" field. This can avoid user confusion and reduce the chance of "required" input error.
-- There may be cases where optional fields may be appropriate. In these forms, clearly denote which fields are optional vs. required. This way, users won't be forced to infer anything - but can instead stay focused on completing their task.
-- Limit the number of optional fields as much as possible. Remember that the more fields you have on your form, the more you risk overwhelming the users.
+**Format and fields**
+- User-entered input must be encoded to prevent [SQL injection](https://www.w3schools.com/sql/sql_injection.asp) and [XSS attacks](https://en.wikipedia.org/wiki/Cross-site_scripting).
+- Wrap all input elements within a FormField `<FormField label="Username"><TextInput /></FormField>`  
+- Reveal fields progressively so the user is not overwhelmed 
```

Comment:
```suggestion
- Reveal fields progressively so the user is not overwhelmed .
```
We have been following the convention of using a `.` at the end of bullet points. Should we update all bullets here to be consistent